### ISSUE TITLE
Upstream changes?

### ISSUE THREAD

**Issue by catskul** [kdab-hotspot/hotspot-perfparser#1]
Any chance you'll be upstreaming any of your changes to the parent project?

**Comment by dmikushin** [comment]
Dear @catskul I'm currently in the middle of introducing MacOS support. Are you interested in any particular change?

**Comment by catskul** [comment]
Nope, I'm just currently contributing back to the parent repo and running into issues (in Linux) and hoping to get any incidental improvements that might come from your MacOS ports.

No rush, just wanted to demonstrate interest to let you know your work is appreciated.

**Comment by dmikushin** [comment]
oh thanks!
